stories/datepicker/datepicker.stories.js

```diff
@@ -0,0 +1,1684 @@
+import '../../dist/calendar.css';
+
+export default {
+    title: 'Patterns/Datepicker',
+    parameters: {
+        description: `The date picker lets users select a localized date using touch, mouse, or keyboard input. It consists of two parts: the date input field and the date picker.
+
+Use this control if the user needs to enter a single date or a date range. The control also allows users to navigate directly from one month or year to another.
+
+The date-picker component is an opinionated composition of the <code>input-group</code>, <code>popover</code> and <code>calendar</code> components to accomplish the UI pattern for picking a date.
+
+This component mostly relies on the CSS of other components and has very little CSS of its own.`,
+        tags: ['f3', 'a11y', 'theme']
+    }
+};
+
+export const sizes = () => `
+<div class="fd-docs-date-picker">
+	<div class="fd-popover">
+		<div class="fd-popover__control">
+            <label class="fd-form-label" for="asdewq123">Pick a date:</label>
+            <div class="fd-input-group">
+				<input class="fd-input fd-input-group__input" type="text" value="" placeholder="Pick a date" id="asdewq123">
+				<span class="fd-input-group__addon fd-input-group__addon--button">
+				<button class="fd-input-group__button fd-popover__control fd-button fd-button--transparent sap-icon--appointment-2" aria-controls="Itcgq828" aria-expanded="false" aria-haspopup="true" aria-label="button"></button>
```

Comment:
I don't think the aria-label can just be "button" - why did we move away from aria-hidden?

Reply:
It shouldn't be aria-hidden since it's a focusable/actionable button. We do need a better aria-label for it than `button` though.

Reply:
The site https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/datepicker-dialog.html uses `aria-label="Choose Date"`

Reply:
updated the label